File: Proyecto-Prog_Avanzada-railway/src/main/java/co/edu/uniquindio/proyecto/proyectotienda/dto/ComentarioGetDTO.java
package co.edu.uniquindio.proyecto.proyectotienda.dto;

import lombok.*;

import java.time.LocalDateTime;


@AllArgsConstructor
@Getter
@Setter
public class ComentarioGetDTO {
    private int codigo;
    private LocalDateTime fecha;
    private String mensaje;
    private int codigoUsuario;
    private int codigoProducto;
}

File: Proyecto-Prog_Avanzada_Frontend-master/src/app/modelo/comentario-get-dto.ts
export class ComentarioGetDTO {
    codigo: number = 0;
    fecha: Date;
    mensaje: string = "";
    codigoUsuario: number = 0;
    codigoProducto: number = 0;

    constructor(codigo: number, fecha: Date, mensaje: string, codigoUsuario: number, codigoProducto: number) {
        this.codigo = codigo;
        this.fecha = fecha;
        this.mensaje = mensaje;
        this.codigoUsuario = codigoUsuario;
        this.codigoProducto = codigoProducto;
    }
}
